dist/index.es.js

```diff
@@ -19,50 +24,51 @@ function ConfettiGenerator(params) {
 
   //////////////
   // Setting parameters if received
-  if(params) {
-    if(params.target)
+  if (params) {
+    if (params.target)
       appstate.target = params.target;
-    if(params.max)
+    if (params.max)
       appstate.max = params.max;
-    if(params.size)
+    if (params.size)
       appstate.size = params.size;
-    if(params.animate !== undefined && params.animate !== null)
+    if (params.animate !== undefined && params.animate !== null)
       appstate.animate = params.animate;
-    if(params.respawn !== undefined && params.respawn !== null)
+    if (params.respawn !== undefined && params.respawn !== null)
       appstate.respawn = params.respawn;
-    if(params.props)
+    if (params.props)
       appstate.props = params.props;
-    if(params.colors)
+    if (params.colors)
       appstate.colors = params.colors;
-    if(params.clock)
+    if (params.clock)
       appstate.clock = params.clock;
-    if(params.start_from_edge !== undefined && params.start_from_edge !== null)
+    if (params.start_from_edge !== undefined && params.start_from_edge !== null)
       appstate.start_from_edge = params.start_from_edge;
-    if(params.width)
+    if (params.width)
       appstate.width = params.width;
-    if(params.height)
+    if (params.height)
       appstate.height = params.height;
-    if(params.rotate !== undefined && params.rotate !== null)
+    if (params.rotate !== undefined && params.rotate !== null)
       appstate.rotate = params.rotate;
   }
 
   //////////////
   // Properties
-  var cv = document.getElementById(appstate.target);
+  var cv = (appstate.target instanceof HTMLCanvasElement) ? appstate.target : document.getElementById(appstate.target);
```

Comment:
This is the main/only change. Rest of the changes are just beautify re-indenting the code.